examples/non-examples/bad-lexical-env.golden

```diff
@@ -0,0 +1 @@
+Unknown: #[bad-lexical-env.kl:3.24-3.25]<y>
```

Comment:
this should succeed, no? g is a function which returns another function, and that seems fine.